packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -842,11 +841,16 @@ class PlatformViewLink extends StatefulWidget {
 class _PlatformViewLinkState extends State<PlatformViewLink> {
   int? _id;
   PlatformViewController? _controller;
+  bool _platformViewCreated = false;
   Widget? _surface;
   FocusNode? _focusNode;
 
   @override
   Widget build(BuildContext context) {
+    if (!_platformViewCreated) {
+      // Depending on the platform, the initial size can be used to size the platform view.
+      return _PlatformViewPlaceHolder(onLayout: (Size size) => _controller!.create(size: size));
```

Comment:
how are we sure _controller is not null here?

Reply:
looks like there are some other places in the old that directly using controller!. It is not in the scope of this PR but it seems unsafe. Could you just fix it for this current line? maybe something like:
```dart
assert(_controller != null);
if (_controller == null) {
  return SizedBox(size)
}
return _PlatformViewPlaceHolder(onLayout: (Size size) => _controller!.create(size: size));
```

Reply:
done